Fix Functional Geb Tests

Error-1 in spinner stops spinning:

`ERROR org.grails.web.errors.GrailsExceptionResolver - IllegalArgumentException occurred when processing request: [GET] /home/getRoute - parameters:
start: Leipzig
destination: Leipzig
additionalTravelTime: 1800
lang: de
The start-position equals the destination-position.. Stacktrace follows:
java.lang.reflect.InvocationTargetException: null
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: The start-position equals the destination-position.
	at trip.planner.osm.api.POIApi.validateCoords(POIApi.groovy:69)
	at trip.planner.osm.api.POIApi.<init>(POIApi.groovy:31)
	at trip.planner.HomeController.getPOIs(HomeController.groovy:98)
	at trip.planner.HomeController.getRoute(HomeController.groovy:43)
	... 3 common frames omitted`

Error-2 in no route exists:

`org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Error: Unable to find 'Zoo Leipzig, Pfaffendorfer Straße 29 Leipzig Deutschland'!}
  (Session info: chrome=50.0.2661.75)
  (Driver info: chromedriver=2.21.371461 (633e689b520b25f3e264a2ede6b74ccc23cb636a),platform=Linux 3.19.0-32-generic x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2 milliseconds: null`

--- a/src/integration-test/groovy/trip/planner/HomeFunctionalSpec.groovy
+++ b/src/integration-test/groovy/trip/planner/HomeFunctionalSpec.groovy
@@ -55,25 +55,23 @@ class HomeFunctionalSpec extends GebSpec {
         when: "Valid cities are given"
         HomePage page = browser.to HomePage
         page.enterCity(page.startSearchField, LEIPZIG)
-        page.clickFirstAutocomplete(page.startDropdown)
         page.enterCity(page.destinationSearchField, BERLIN)
-        page.clickFirstAutocomplete(page.destinationDropdown)
         page.submitButton.click()
 
         then: "spinner stops spinning"
-        assert waitFor(wait: 150){!page.spinnerExists()}
+        assert waitFor(155){
+            !page.spinnerExists()
+        }
     }
 
     void "test generate route no route exists"() {
         when: "Unconnectable cities are given"
         HomePage page = browser.to HomePage
         page.enterCity(page.startSearchField, LEIPZIG)
-        page.clickFirstAutocomplete(page.startDropdown)
         page.enterCity(page.destinationSearchField, NEW_YORK)
-        page.clickFirstAutocomplete(page.destinationDropdown)
 
         then: "Alert occurs"
-        withAlert(wait: true) {
+        withAlert(wait: 151) {
             page.submitButton.click()
         }
     }

--- a/src/integration-test/groovy/trip/planner/HomePage.groovy
+++ b/src/integration-test/groovy/trip/planner/HomePage.groovy
@@ -34,11 +34,6 @@ class HomePage extends Page {
         autoComplNav.children()*.text()
     }
 
-    void clickFirstAutocomplete(Navigator autoComplNav){
-        waitFor(10) { autoComplNav.displayed }
-        autoComplNav.click()
-    }
-
     boolean spinnerExists() {
         spinner
     }

